src/components/index.ts

```diff
@@ -74,6 +74,8 @@ export type { PopoverProps } from "./Popover/Popover"
 export { default as Radio } from "./Radio/Radio"
 export type { RadioProps } from "./Radio/Radio"
 
+export { default as SkeletonLoader } from "./SkeletonLoader/SkeletonLoader"
```

Comment:
S: We should also export the `SkeletonLoaderProps` type, as we do on the other components, in case the library user needs it